### PR TITLE
Remove older policy attach behavior for LDAP

### PR DESCRIPTION
**Pull request by donatello**
## Description
This removes previously deprecated behavior.

`mc admin policy attach` will now raise an error when the user does not exist. Previously it was allowed to have a non-existent user to attach a policy to support setting policies for LDAP authenticated users.

Now, we have `mc idp ldap policy attach` for this purpose which validates if the user exists in the configured LDAP directory.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

File: cmd/admin-handlers-users.go [hunk 1]
```diff
@@ -1767,7 +1767,7 @@ func (a adminAPIHandlers) ListPolicyMappingEntities(w http.ResponseWriter, r *ht
 	writeSuccessResponseJSON(w, econfigData)
 }
 
-// AttachPolicyBuiltin - POST /minio/admin/v3/idp/builtin/attach
+// AttachPolicyBuiltin - POST /minio/admin/v3/idp/builtin/policy/attach
```
Comment (shtripat) [inline cmd/admin-handlers-users.go:1770]: Is it like API end point was always like this `POST /minio/admin/v3/idp/builtin/policy/attach`? Just correction in comment?

Comment (donatello) [reply cmd/admin-handlers-users.go:1770]: Yes here it's just a correction of the comment

**Review by shtripat — APPROVED** [review]
lgtm

**Review by harshavardhana — APPROVED** [review]

**Review by vadmeste — APPROVED** [review]
LGTM

**Comment by harshavardhana** [comment]
Please fix the conflicts @donatello

**Pull request merged**